skellytracker/SINGLE_IMAGE_RUN.py

```diff
@@ -1,10 +1,22 @@
 import cv2
 from pathlib import Path
 
-from skellytracker.trackers.bright_point_tracker.brightest_point_tracker import BrightestPointTracker
+from skellytracker.trackers.bright_point_tracker.brightest_point_tracker import (
+    BrightestPointTracker,
+)
 from skellytracker.trackers.charuco_tracker.charuco_tracker import CharucoTracker
-from skellytracker.trackers.mediapipe_tracker.mediapipe_holistic_tracker import MediapipeHolisticTracker
-from skellytracker.trackers.yolo_tracker.yolo_tracker import YOLOPoseTracker
+
+try:
+    from skellytracker.trackers.mediapipe_tracker.mediapipe_holistic_tracker import (
+        MediapipeHolisticTracker,
+    )
+except:
+    print("To use mediapipe_holistic_tracker, install skellytracker[mediapipe]")
```

Comment:
```suggestion
    print("\n\nTo use mediapipe_holistic_tracker, install skellytracker[mediapipe]\n\n")
```

Make it easier for users to pick the message out of the terminal output